### PR TITLE
Update BuildingFromSource.md

### PR DESCRIPTION
**Pull request by PromanSEW**
# Pull Request Checklist

* [X] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
* [X] Have you read through the [contributor guidelines](https://www.playframework.com/contributing)?
* [X] Have you signed the [Lightbend CLA](https://www.lightbend.com/contribute/cla)?
* [ ] Have you referenced any issues you're fixing using [commit message keywords](https://help.github.com/articles/closing-issues-using-keywords/)?
* [ ] Have you added copyright headers to new files?
* [ ] Have you checked that both Scala and Java APIs are updated?
* [X] Have you updated the documentation for both Scala and Java sections?
* [ ] Have you added tests for any changed functionality?

### CODE REVIEW

File: documentation/manual/hacking/BuildingFromSource.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ From the shell, first checkout the Play source:
 $ git clone git://github.com/playframework/playframework.git
 ```
 
-Checkout the branch you want, the current development branch is called `master`, while stable branches for major releases are named with a `.x`, for example, `2.5.x`.
+Checkout the branch you want, the current development branch is called `master`, while stable branches for major releases are named with a `.x`, for example, `2.7.x`.
 
 Now go to the `framework` directory and run `sbt`:
```
Comment (marcospereira) [inline documentation/manual/hacking/BuildingFromSource.md]: This is not necessary anymore. Can you also rewrite this?

Comment (marcospereira) [inline documentation/manual/hacking/BuildingFromSource.md]:
```suggestion
Now run `sbt`:
```

**Comment by PromanSEW** [comment]
What about backporting this to `2.7.x`?

**Comment by dwijnand** [comment]
2.7.x 448f9e9c3

**Pull request merged**